spread-shellcheck

```diff
@@ -170,13 +172,13 @@ def checkpath(loc):
         raise ShellcheckFailures(failures=failed)
 
 
-def loadmusts(mustpath):
-    musts = set()
-    with open(mustpath) as mf:
-        for line in mf.readlines():
+def loadfilelist(flistpath):
+    flist = set()
+    with open(flistpath) as inf:
+        for line in inf.readlines():
```

Comment:
You don't need readlines here, using the default iterator will be slightly more memory conservative as it will keep only one line in memory at a time.